ui.h

```diff
@@ -229,8 +229,12 @@ _UI_EXTERN void uiRadioButtonsSetSelected(uiRadioButtons *r, int n);
 _UI_EXTERN void uiRadioButtonsOnSelected(uiRadioButtons *r, void (*f)(uiRadioButtons *, void *), void *data);
 _UI_EXTERN uiRadioButtons *uiNewRadioButtons(void);
 
+struct tm;
 typedef struct uiDateTimePicker uiDateTimePicker;
 #define uiDateTimePicker(this) ((uiDateTimePicker *) (this))
+_UI_EXTERN void uiDateTimePickerTime(uiDateTimePicker *d, struct tm *time);
```

Comment:
Should probably add a TODO comment that the documentation should explain what happens with `tm_wday`, `tm_yday`, and `tm_isdst` (which I am leaving up to you if you have a better idea than [my suggestion](https://github.com/andlabs/libui/issues/295#issuecomment-366853845), since your code is inconsistent about this).

Also starting to lean toward including `<time.h>` instead of declaring `struct tm`, but I'll leave that up for later.

Reply:
`tm_wday` and `tm_yday` will be undefined, and `tm_isdst` will always be -1 for now. I will document them as such. 

I believe `<time.h>` is fairly portable here, but nothing really wrong with that single forward declaration..

Reply:
Fine by me 👍